settings/PandoraSettings_TestBeam_ProtoDUNE.xml

```diff
@@ -317,11 +317,21 @@
     <algorithm type = "LArNeutrinoProperties">
         <NeutrinoPfoListName>NeutrinoParticles3D</NeutrinoPfoListName>
     </algorithm>
-    
-    <!-- Track and shower building --> 
-    <algorithm type = "LArTrackParticleBuilding"> 
-        <PfoListName>TrackParticles3D</PfoListName> 
-        <VertexListName>DaughterVertices3D</VertexListName> 
+
+    <!-- Track and shower building -->
+    <algorithm type = "LArTrackParticleBuilding">
+        <PfoListName>TrackParticles3D</PfoListName>
+        <VertexListName>DaughterVertices3D</VertexListName>
+    </algorithm>
+
+    <algorithm type = "LArTestBeamParticleCreation">
+        <PfoListName>NeutrinoParticles3D</PfoListName>
+        <TrackPfoListName>TrackParticles3D</TrackPfoListName>
+        <ShowerPfoListName>ShowerParticles3D</ShowerPfoListName>
+        <VertexListName>NeutrinoVertices3D</VertexListName>
+        <DaughterVertexListName>DaughterVertices3D</DaughterVertexListName>
+        <KeepStartVertex>false</KeepStartVertex>
```

Comment:
Suggest removing these to try to make XML more minimal, ensuring that C++ contains expected default values.